examples/UKCore-Specimen-SampleState-Example.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Specimen xmlns="http://hl7.org/fhir">
+ <id value="UKCore-Specimen-SampleState-Example"/>
+ <status value="available"/>
```

Comment:
might be worth putting the extension first, and pretty printing this, I've just looked at it and thought, is SampleState an extension on status?